### PR TITLE
BRIEFCASE-196: Logging section in the contribution guide.

### LINKED ISSUE #196: Describe logging in contribution guide
Briefcase uses [Apache Commons Logging](https://commons.apache.org/proper/commons-logging/) to allow for users to use different logging implementations and configurations. This is confusing and hard to figure out for those not familiar with Java logging. The contribution guide should include a section on logging. [Spring Boot](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-logging.html) has some sections that are relevant and good models.

### PR DESCRIPTION
**Pull request by nribeka**
Closes #196 

#### Are there any risks to merging this code? If so, what are they?
No risk since this is just documentation changes.

This is assuming #218 is approved.

### CODE REVIEW

**Comment by nribeka** [comment]
Please review the grammar and if I need to add more info to it :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,6 +63,48 @@ Small fixes that target very particular bugs may occasionally be merged without
...
+* Log Level (ERROR, WARN, INFO, or DEBUG)
+* Thread name
+* Logger name (usually the source class name)
+* The log message
+
+### Console output
+Console output will have the same format as above but will default to only display INFO, WARN and ERROR log messages.
+
+### File output
+In addition to the console output, briefcase will also write the log mesages to file. The file will have all messages
```
Comment (kkrawczyk123) [inline CONTRIBUTING.md]: One "s" is missing in the first messages word.

**Comment by lognaturel** [comment]
To be clear, this is on hold for now. All of the digging into logging has made it clear that there are deeper issues to address and @ggalmazor is doing some re-architecting. Once there's a dedicated command line interface that doesn't use logging we will re-assess the best defaults for logging.

**Comment by yanokwa** [comment]
Closing because this PR is no longer relevant since https://github.com/opendatakit/briefcase/pull/381 and https://github.com/opendatakit/briefcase/pull/356 have been merged.

**Pull request closed without merging**